src/cljs/main/broadfcui/page/workspace/summary/synchronize.cljs

```diff
@@ -25,15 +26,18 @@
             [comps/Blocker {:banner "Granting permission..."}])
           (when owned-methods
             [:div {}
+             (messages/get-methods-repo-group-alert)
              "In order to allow the users you have added to this workspace to run the methods configured
               for it, you must grant them access to the following:"
              [:ul {} (map (fn [method] [:li {} (sync-common/get-method-display (:method method))]) owned-methods)]])
           (when unowned-methods
             [:div {}
              (if owned-methods
-               "Additionally, users will need to request access to the following methods:"
-               "In order to run the methods configured for this workspace, users will need to request
-                access to the following:")
+               [:div {}
+                (messages/get-methods-repo-group-alert)
+                "Additionally, users will need to request access to the following methods:"
+                "In order to run the methods configured for this workspace, users will need to request
```

Comment:
This `if` is broken now.

Reply:
omg it took me a half hour to see why I broke it. fixing...